akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -78,7 +77,12 @@ object CircuitBreaker {
  * @param resetTimeout [[scala.concurrent.duration.FiniteDuration]] of time after which to attempt to close the circuit
  * @param executor [[scala.concurrent.ExecutionContext]] used for execution of state transition listeners
  */
-class CircuitBreaker(scheduler: Scheduler, maxFailures: Int, callTimeout: FiniteDuration, resetTimeout: FiniteDuration)(implicit executor: ExecutionContext) extends AbstractCircuitBreaker {
+class CircuitBreaker(
+  scheduler:    Scheduler,
+  maxFailures:  Int,
+  callTimeout:  FiniteDuration,
+  resetTimeout: FiniteDuration,
+  factor:       Int            = 1)(implicit executor: ExecutionContext) extends AbstractCircuitBreaker {
```

Comment:
Good catch, will add require statement. However, I don't think `Double` is necessary.